### PR TITLE
Fix handshake w/lab

### PR DESCRIPTION
**Pull request by jennyf19**
The lab was not accepting the handshake, so added in ServicePointManger.

### CODE REVIEW

File: msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs [hunk 1]
```diff
@@ -42,6 +42,12 @@ public class UsernamePasswordIntegrationTests
         public const string Authority = "https://login.microsoftonline.com/organizations/";
         public string[] Scopes = { "User.Read" };
         AuthHelper authHelper = new AuthHelper();
+        
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;
```
Comment (henrik-me) [inline msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs]: >SecurityProtocolType.Ssl3 [](start = 80, length = 25)

Can we remove this one? It should only be using TLS 1.2+ #Resolved

**Review by henrik-me — CHANGES_REQUESTED** [review]
:clock1:

File: msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs [hunk 2]
```diff
@@ -42,6 +42,12 @@ public class UsernamePasswordIntegrationTests
         public const string Authority = "https://login.microsoftonline.com/organizations/";
         public string[] Scopes = { "User.Read" };
         AuthHelper authHelper = new AuthHelper();
+        
+        [TestInitialize]
+        public void TestInitialize()
```
Comment (bgavrilMS) [inline msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs]: >TestInitialize [](start = 20, length = 14)

It seems to me that you want to run this code once for all tests in this assembly? In that case, note that there are 2 other init attributes in mstest - ClassInitialize and AssemblyInitialize. The last seems more appropriat for your needs: [https://stackoverflow.com/questions/2382552/is-it-possible-to-execute-code-once-before-all-tests-run](https://stackoverflow.com/questions/2382552/is-it-possible-to-execute-code-once-before-all-tests-run) #Resolved

**Review by bgavrilMS — APPROVED** [review]
:shipit:

File: msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs (hunk 2 above)
Comment (jennyf19) [reply msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs]: very cool. You're right, i only need this once for the class, so I changed to classInitialize. Thanks for the suggestion. 

---
In reply to: [221246985](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1281#discussion_r221246985) [](ancestors = 221246985)

File: msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs (hunk 1 above)
Comment (jennyf19) [reply msal/tests/Test.MSAL.NET.Integration/UsernamePasswordIntegrationTests.cs]: had some issues w/keyvault, so this took a little longer to fix.

---
In reply to: [221143624](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1281#discussion_r221143624) [](ancestors = 221143624)

**Review by henrik-me — APPROVED** [review]
:shipit:

**Pull request merged**